### ISSUE TITLE
Unable to post ad with image

### ISSUE THREAD

**Issue by thanksforkijijimanager** [jackm/kijiji-manager#23]
Been using this Kijiji-manager for quite a long time and very thankful for it :)
Recently have not been able to post new ads that contain pictures
Still able to repost existing ads and post ads without images
Tried updating from old version to 1.5 to 1.7 but still no luck
Tried to make a secret key for the cfg file but still no luck
I am able to reproduce the issue
After pressing post ad, payload is not created, ad is not posted and I get the following error:
KijijiApiException: Error: Auth token is invalid. Validation of the authentication token in API request failed.
Issue persists regardless of category and Kijiji account used

**Comment by prgrm** [comment]
Yes same issue here.

**Comment by thanksforkijijimanager** [comment]
Issue is still on-going

**Comment by rybodiddly** [comment]
the eBay auth token required for picture uploads was recently updated. To fix the issue, edit kijijiapi.py line 240:
```
# eBay auth token used by Kijiji
        token = 'AgAAAA**AQAAAA**aAAAAA**y9arYQ**nY+sHZ2PrBmdj6wVnY+sEZ2PrA2dj6MHloakAZGGqQSdj6x9nY+seQ**j8wBAA**AAMAAA**XCyjvCgYtZFhE1DSKsy4Fhz8o9eiKRNMtKr/P1ltxsz2B9XZgQOhlI0lG8NUmFbnltvBo5GsIYrQ56N/lZW8/yzmUczzl87c+iDF1GBH+La4WuDnGOTMlg2gLbroX7gw87xBvRaaUPRP7ZaLzhO6MRbGrTByZf/C+VphIZ5tzsUHJvxvjsgAuEaQ/vTvNfrd23FKJ8CgBt7tcK0YEJM0uBwgTzcACe2XnUKZF2f5C8AeS6z5vH5U9rf7r0CZ6BG5o2sD9umOJYCm9VimQbI/7+FCQNsqcV8erS1sng9/0fIHVk9wgfYF0yTWn0Qn8Q/Vc3jBbIAPqWKRlm2tSdjYwV8hVRNOhe7EfyHt+cpPpU6GwngeZc3+lH4eyl83443wlbRv3IWuH+eB9rdhRg1J1UIfBN5R4O1CdHPDLzRBBgduMTSB5RVHJP4cEFQL2uycW2LJq6HAFcqt3qW2mF6HdQZ8nHa3FuN7cESK4hoxJ2JABLAGHwFEDZNDlCr0gFSBZ5ajni92yxUB9h2MkBD2gMHf8qZGqHCIQVNBnEs81Dq+BK0Qc1FLaYtxQaWErTq4nOd76SkKE7uqqK4IRAhJ/hcyeqPBFSe5bNyNtiSE1g9ff0g1KaV9n5Zry6rJEr9/gOunTiTN9qLDpf2sYlD3tJL3cCSsgLCZlpAMt3+YXDmK6Fx1UA5BQAg5UIeWr1QrE4w/YEKfbD3zdf7Ww5/qIhw5trjdqvbh6Z9tHq9YGBu+wUm4AA0hHdMGa7acgU7L'
```

**Comment by thanksforkijijimanager** [comment]
Confirmed worfking!!!! THANKS !!!!!!!!!!

**Comment by jackm** [comment]
Merged in PR #24 

You should pull the latest version from the master branch to use this fix.

**Comment by jackm** [comment]
Pushed new [release v0.1.8](../releases/tag/v0.1.8) which includes this fix.
Also uploaded new release to PyPI so running `pip install --upgrade kijiji-manager` will also update to the latest.
